bc/web/static_src/tailwind.config.js

```diff
@@ -45,8 +45,12 @@ module.exports = {
     ],
     theme: {
       extend: {
+        screens: {
+          'xs': '420px',
```

Comment:
Do we need this? I thought tailwind pretty much came with viewport sizes and you could just use what they ship?

Reply:
We don't need it anymore. I was using this custom breakpoint to change the font size for viewports bigger than 420px but the new responsive design just needs the default breakpoints.